docs/src/architecture/08_concepts/document_templates/proposal/F14-Generic/0ce8ab38-9258-4fbc-a62e-7faa6e58318f.schema.json

```
@@ -489,6 +490,172 @@
                 "open_source"
             ]
         },
+        "campaign_category": {
+            "type": "object",
```

The "top-level" object should be segment.

```suggestion
            "$ref": "#/definitions/segment",
```